portfolio/src/main/webapp/script.js

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Fetches content of /data from the server and adds it to the DOM.
+ */
+function getData() {
+
```

Comment:
According to the JS style guide, you shouldn't have blank lines at the start or end of a function body:

"Blank lines at the start or end of a function body are not allowed."
https://google.github.io/styleguide/jsguide.html#formatting-vertical-whitespace